src/kind_lab_to_nwb/rat_behavioural_phenotyping_2025/utils/utils.py

```diff
@@ -116,3 +118,39 @@ def get_session_ids_from_excel(subjects_metadata_file_path: FilePath, task_acron
     # Remove trailing whitespace from each session ID
     session_ids = [session_id.rstrip() for session_id in session_ids]
     return session_ids
+
+
+def convert_to_mp4(video_file_path):
+    """Convert a video file to .mp4 format using ffmpeg.
+    If the file is not in .mp4 format, it will create a new .mp4 file in a subdirectory called "converted".
+
+    Parameters
+    ----------
+    video_file_path : Path
+        Path to the .ts video file
+
+    Returns
+    -------
+    Path
+        Path to the converted .mp4 video file
+    """
+    # Check if the input file exists
+    if not video_file_path.is_file():
+        raise FileNotFoundError(f"The file {video_file_path} does not exist.")
+
+    # Create a subdirectory called "converted"
+    output_dir = video_file_path.parent / "converted"
+    output_dir.mkdir(parents=True, exist_ok=True)
+    # Define the output file path by replacing the extension with .mp4
+    output_file_path = output_dir / (video_file_path.stem + ".mp4")
+
+    # Check if the output file already exists
+    if output_file_path.is_file():
+        print(f"The file {output_file_path} already exists. Skipping conversion.")
+        return output_file_path
+
+    # Use ffmpeg to convert the video file
+    command = f'ffmpeg -i "{video_file_path}" -c:v copy -c:a aac "{output_file_path}"'
+    os.system(command)
```

Comment:
I would still use `subprocess` here instead of os.system for better error handling and more control, something like this:

```python
import subprocess

try:
    subprocess.run(['ffmpeg', '-i', video_file_path, '-c:v', 'copy', '-c:a', 'aac', output_file_path], 
                  check=True,
                  capture_output=True)
except subprocess.CalledProcessError as e:
    print(f"An error occurred: {e}")
```